input/pages/app-launch.md

```diff
@@ -1,4 +1,4 @@
-The SMART App Launch Framework connects third-party applications to Electronic
+Note that the SMART App Launch Framework connects third-party applications to Electronic
```

Comment:
Why make the intro paragraph a "note"? It's definitional context...

Reply:
The note was already there.  Just trying to be consistent instead of "Note: ... " vs "Note that.... "

Reply:
That is an error,  I don't recall making that edit.   I made all the" Note: ..." to "Note that ..."  for consistency.  n

Reply:
OK, I've fixed it.